recipes/scine-utilities-python/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "utilities" %}
+{% set pkg_name = "scine-" + name + "-python" %}
+{% set version = "4.0.0" %}
```

Comment:
Why are you jinja templating these names when they are each only used once?

Reply:
Simplified this part.